### DISCUSSION TITLE
leastLoad是什么意思？可以给一个配置示例吗？

### DISCUSSION THREAD

**Discussion by draftman9** [v2fly/v2ray-core#2357] [Q&A]
如题。

**Comment by guoyecn** [comment]
通过检查节点的RTT标准偏差，选择RTT标准偏差最小的节点。
RTT标准偏差小，说明该节点的流量负载比较小，节点比较稳定。
配置语法和leastPing完全一样。

**Comment by mkmark** [comment]
https://github.com/v2fly/v2ray-core/pull/589
